### ISSUE TITLE
Audio?

### ISSUE THREAD

**Issue by CaptainLynk** [johnventions/Arduino-Director#1]
I can not figure out how to add audio everything else  works though!

**Comment by johnventions** [comment]
hey! Sorry - the documentation for that isn't very good (aka nonexistant)

Here is what i did:
- Make a folder called "assets" under dist
- Add mp3 files (or whatever file type) to the assets folder
- Set the "audio" field to "/assets/myfilename.mp3"

You may have to hit the refresh button next to the audio input field

**Comment by johnventions** [comment]
Let me know if that works for you! Also, thanks for trying this out, I think you're the first person to do so

**Comment by CaptainLynk** [comment]
of course! the animation works beautifully! it is my first animatronic and
I love it!

**Comment by johnventions** [comment]
Well hot diggity dog that's awesome news. Would love to hear more about it / I have a couple ideas for software improvements. My email is johnventions AT gmail if you want to chat animatronics
